confluence-xml/src/test/resources/confluencexml/img.test

```diff
@@ -0,0 +1,1897 @@
+            </entry>
+            <entry>
+              <string>lastrevision</string>
+              <string>2</string>
+            </entry>
+          </parameters>
+        </p>
+        <wikiDocumentRevision revision="1">
+          <p>
+            <parameters>
+              <entry>
+                <string>revision_author</string>
+                <string>XWiki.alexandru_farcasi@xwiki_com</string>
+              </entry>
+              <entry>
+                <string>revision_date</string>
+                <date>2024-10-04 07:26:38.259 UTC</date>
+              </entry>
+              <entry>
+                <string>revision_comment</string>
+                <string></string>
+              </entry>
+              <entry>
+                <string>title</string>
+                <string>IMG Macro Holder</string>
+              </entry>
+              <entry>
+                <string>content</string>
+                <string>this page will hold the macro calls</string>
+              </entry>
+              <entry>
+                <string>syntax</string>
+                <org.xwiki.rendering.syntax.Syntax>
+                  <type>
+                    <name>XWiki</name>
+                    <id>xwiki</id>
+                    <variants class="empty-list"/>
+                  </type>
+                  <version>2.1</version>
+                </org.xwiki.rendering.syntax.Syntax>
+              </entry>
+            </parameters>
+          </p>
+        </wikiDocumentRevision>
+        <wikiDocumentRevision revision="2">
+          <p>
+            <parameters>
+              <entry>
+                <string>revision_author</string>
+                <string>XWiki.alexandru_farcasi@xwiki_com</string>
+              </entry>
+              <entry>
+                <string>revision_date</string>
+                <date>2024-10-04 07:45:19.150 UTC</date>
+              </entry>
+              <entry>
+                <string>revision_comment</string>
+                <string></string>
+              </entry>
+              <entry>
+                <string>title</string>
+                <string>IMG Macro Holder</string>
+              </entry>
+              <entry>
+                <string>content</string>
+                <string>this page will hold the macro calls
+
+~=[[image:url:https://cdn.pixabay.com/photo/2022/09/27/19/46/ai-generated-7483596_960_720.jpg||alt="some alt text" dir="ltr" title="A nice title" longdesc="A very beautiful image" width="69px" style="opacity: 0.5" id="someId" lang="ro" class="someClass" usemap="someid" height="420px"]][[image:attach:~~7120200cbb2ffd37954f0ba8329a7fefd47b8e.IMG Macro Holder.IMG Macro attachment holder.WebHome@270378865_3175061552717005_4796137940594548712_n.jpg||queryString="api=v2" alt="some alt text" dir="ltr" title="A nice title" longdesc="A very beautiful image" width="69px" style="opacity: 0.5" id="someId" lang="ro" class="someClass" usemap="someid" height="420px"]]</string>
```

Comment:
> In the [XWiki syntax documentation](https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/XWikiSyntax/?syntax=2.1&section=Images), the longdesc attribute is not explicitly specified. However, when testing, I found that the attribute is handled and transformed into data-xwiki-translated-attribute-longdesc

I would not call that "handled" :) This essentially mean "no idea what is this attribute is, looks fishy, so I'm going to put it in a data-* thingy just to not loose it".

Reply:
Thanks for the clarification, @tmortagne! Since this is more of a fallback, do you think we should remove the longdesc attribute entirely? The longdesc attribute was supposed to hold a link to a page with more information about the image, but it is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/longDesc).

Reply:
Honestly, I don't know what's the best from Confluence import point of view. It basically all depends if it's important to keep this data or not, if it's not then I would remove it as it's going to be noise on XWiki side.

Reply:
I will keep it, users might expect to still have that data

Reply:
Hmm, what about setting it as `title` instead? At least this is actually something, it depends if it's a problem to get a tooltip with this or not.